frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/validation-utils.ts

```diff
@@ -351,9 +406,8 @@ const taskValidation = (formValues: PipelineBuilderFormYamlValues, taskType: Tas
                   'is-workspace-link-broken',
                   i18n.t('pipelines-plugin~Workspace name has changed, reselect'),
                   (workspaceValue?: string) =>
-                    !!workspaceValue && !!workspaces.find(({ name }) => name === workspaceValue),
-                )
-                .required(i18n.t('pipelines-plugin~Required')),
+                    !workspaceValue || !!workspaces.find(({ name }) => name === workspaceValue),
+                ),
```

Comment:
This here was the other latest fix. This checks passes now if there is no value is defined.

The tests `are-workspaces-available` and `is-workspace-link-broken` only handle cases where a workspaceValue is defined. We have the `is-workspace-is-required` to verify that the user has any workspace defined (for required fields).